src/models/TableModel/impl/InMemoryTable.ts

```diff
@@ -143,6 +145,15 @@ export const insertRow = (
   table: Table,
   idRowPair: [IdType, Record<AttributeName, ValueType>],
 ): Table => {
+  // const tableCols = table.columns
```

Comment:
Is this code going to be used in the future?  If not I think this should be deleted.

Reply:
Yeah, I forget to delete these line.